app/views/user/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render 'partial/user_profile', user: @user %>
+
+<div class="user_profile">
+    <div class="user_bio">
+      <h3>Bio</h3>
+      <p class="bio">
+        <b><%= @user.bio %></b><br/><br/>
+        <%= @user.name %>, a passionate food enthusiast, has embarked on a lifelong journey of exploring the world of flavors and cuisines. 
+        His insatiable curiosity and love for all things culinary have led him on a delectable adventure through the vast and diverse realm of food. 
+        Born and raised in a small town, <%= @user.name %>'s fascination with food began at a tender age. 
+        In his family's cozy kitchen, he watched his grandmother whip up traditional recipes with love and skill. 
+        Those moments sparked a culinary fire within him that would never cease to burn.

```

Comment:
[OPTIONAL]
Kindly consider removing the hard-coded bios of the users of the app.
The bio of users should be in the database, and the app should render the bio from the database and not have it hardcoded.